src/intro/setup.md

```diff
@@ -17,16 +17,60 @@ For the rest of the tutorial, we assume that you have Godot 4 installed and avai
 - in your `PATH` as `godot4`,
 - or an environment variable called `GODOT4_BIN`, containing the path to the Godot executable.
 
+
+### Godot from pre-built binaries
+
 Binaries of Godot 4 can be downloaded [from the official website][godot-download].
+For beta and older versions, you can also check the [Archive][godot-download-archive].
+
+
+### Installing Godot via command-line
+
+```bash
+#-- Linux --#
+# For Ubuntu or Debian-based distros
+apt install godot
+
+# For Fedora/RHEL
+dnf install godot
+
+# Distro-independent through Flatpak
+flatpak install flathub org.godotengine.Godot
+
+
+#-- Windows --#
+# Windows installations can be made through WinGet
+winget install --id=GodotEngine.GodotEngine -e
+
+
+#-- macOS --#
+brew install godot
+```
 
+```admonish note
 If you plan to target Godot versions different from the latest stable release, please read [Compatibility and stability][compatibility].
+```
 
 
 ## Rust
 
-[rustup] is the recommended way to install the Rust toolchain, including the compiler, standard library and Cargo (the package manager).
+[rustup][rustup] is the recommended way to install the Rust toolchain, including the compiler, standard library and Cargo (the package manager).
```

Comment:
No need to repeat the link if the referred-to identifier already matches the one in `[ ]` (that's why it wasn't done before).